USAGE.md

```diff
@@ -2,112 +2,129 @@
 
 ## Getting started
 
-### Installing TiKV
+### System requirement
 
-TiKV depends on Etcd and PD, which both need [*Go*](https://golang.org/) installed on your machine (version 1.5+ is required). 
++ Linux or Mac OSX.
++ Rust, nightly version is required.
++ Go, 1.5+ is required.
 
-It is required to use the latest etcd version which supports gRPC from the master branch.
+### Installing TiKV
 
-Etcd:
++ RocksDB
 
-```sh
-git clone https://github.com/coreos/etcd.git ${GOPATH}/src/github.com/coreos/etcd
-cd ${GOPATH}/src/github.com/coreos/etcd
-./build
-```
+    ```sh
+    curl -L https://github.com/facebook/rocksdb/archive/rocksdb-4.3.1.tar.gz -o rocksdb.tar.gz
+    tar xf rocksdb.tar.gz 
+    cd rocksdb-rocksdb-4.3.1 
+    make shared_lib 
+    make install-shared
```

Comment:
Would install-shared create the soft link of the .so files of rocksdb to LD_LIBRARY_PATH or LIBRARY_PATH? Because rustc will find these under LIBRARY_PATH.